scripts/docker-compose.yml

```diff
@@ -16,19 +16,13 @@ services:
       - "8321:8321"
     volumes:
       - ${MONTAGU_ORDERLY_PATH}:/orderly
-    logging:
```

Comment:
This is a script used to bring up test dependencies. Not sure why these were in here, removed as it's helpful for debugging to have logs